recipes/guile/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "guile" %}
+{% set version = "2.2.4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://ftp.gnu.org/gnu/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 33b904c0bf4e48e156f3fb1d0e6b0392033bd610c6c9d9a0410c6e0ea96a3e5c
+  # patches:
+  #   - no_posix_test.patch  # [linux]
```

Comment:
Can you either remove this comment (and the patch) or explain in a comment under what conditions it should be uncommented? Thanks!